package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperterm-summon",
-  "version": "2.0.9",
+  "version": "2.0.10",
```

Comment:
Please revert this change. I'll handle a version bump with the release command for this package.